panopticon/datadog.py

```diff
@@ -57,6 +99,8 @@ def stats(cls):
         This will return a `mock.Mock` instance if the `DATADOG_ENABLED` setting
         is `False`. This makes it possible to run this in development without
         having to make any additional changes or conditional checks.
+
+        :return datadog.ThreadState
```

Comment:
Nope, that works too, but I didn't see any definitive format used in this project. I'll convert...